modules/tutorials/pages/java-tutorial/add-course-enrollment.adoc

```diff
@@ -0,0 +1,143 @@
+= Add Course Enrollments
+:description: Add enrollment information to the student records using the Couchbase SDK.
+:page-topic-type: tutorial
+:page-pagination: full
+:page-toclevels: 2
+:imagesdir: ../../images
+:page-aliases: adding-course-enrollments.adoc
+
+[abstract]
+{description}
+
+== Add Enrollment Details
+
+To add enrollment details to a student record:
+
+. In your `java` directory, create a new file called `AddEnrollments.java`.
+. Paste the following code block into your `AddEnrollments.java` file:
++
+[source, java]
+----
```

Comment:
I'd recommend putting this entire code sample into an `examples` folder at the same level as `images`, `pages`, etc., and pulling it in as an `include`, instead.